### PR TITLE
fix typos: double spaces

### PR DESCRIPTION
**Pull request by OnkarRuikar**
Markdownlint doesn't flag/fix double spaces in text content.
The PR fixes them manually.

**Note:** 
- This will be fixed for good with https://github.com/mdn/content/pull/20824

### CODE REVIEW

**Comment by Josh-Cena** [comment]
Note: these are fixable by Prettier

**Comment by OnkarRuikar** [comment]
@Josh-Cena there is no harm in landing this.
Also, Prettier is [not going to get integrated soon](https://github.com/mdn/content/pull/20537#issuecomment-1243529554). May be not even in Q4.

**Comment by Josh-Cena** [comment]
True, I'm not showing intent to block this or to suggest it's unnecessary. Just saying in the future it could be automated without all the extra work.

File: files/en-us/web/javascript/reference/global_objects/array/grouptomap/index.md [hunk 1]
```diff
@@ -149,12 +149,12 @@ For this reason it is important that anything that needs to use the map keeps a
 
 ```js
 // The key can be modified and still used
-restock['fast']  = true ;
+restock['fast'] = true ;
```
Comment (Josh-Cena) [inline files/en-us/web/javascript/reference/global_objects/array/grouptomap/index.md:152]:
```suggestion
restock['fast'] = true;
```

File: files/en-us/web/javascript/reference/global_objects/date/setdate/index.md [hunk 2]
```diff
@@ -47,7 +47,7 @@ If a negative number is provided for `dayValue`, the date will be set counting b
 ### Using setDate()
 
 ```js
-const theBigDay  = new Date(1962, 6, 7, 12); // noon of 1962-07-07 (7th of July 1962,  month is 0-indexed)
+const theBigDay  = new Date(1962, 6, 7, 12); // noon of 1962-07-07 (7th of July 1962, month is 0-indexed)
```
Comment (Josh-Cena) [inline files/en-us/web/javascript/reference/global_objects/date/setdate/index.md:50]:
```suggestion
const theBigDay = new Date(1962, 6, 7, 12); // noon of 1962-07-07 (7th of July 1962, month is 0-indexed)
```

**Review by Josh-Cena — APPROVED** [review]
Let's get this in! We've all been there: I-am-Prettier...

**Pull request merged**